AUTHORS

```diff
@@ -109,6 +114,7 @@ Caleb Spare <cespare@gmail.com>
 Calen Pennington <cale@edx.org>
 Cameron Boehmer <cameron.boehmer@gmail.com>
 Carl X. Su <bcbcarl@gmail.com>
+cc272309126 <cc272309126@gmail.com>
```

Comment:
actually I found him here https://github.com/docker/docker/commit/1bb02117db80e75f406f6c63d8d50680c1569019
I'm updating the `.mailmap` again